stage #3/java/module #2. REST API Basics/task02/repository/src/main/java/by/epam/esm/enity/Entity.java

```diff
@@ -0,0 +1,21 @@
+package by.epam.esm.enity;
+
+import java.io.Serializable;
+
+/**
+ * class Entity
+ * base abstract entity class
+ * @author Aliaksei Tkachuk
+ * @version 1.0
+ */
+public abstract class Entity implements Serializable {
```

Comment:
Please explain me the purpose of Serializable interface here, how does it help you in your current app?